src/Entities/Registry.cpp

```diff
@@ -148,26 +148,25 @@ void Registry::PrepareDrawDescs(bool drawBoundingBox)
 		renderCtx.instancedDrawDescs.emplace(std::piecewise_construct, std::forward_as_tuple(meshId), std::forward_as_tuple(offset, count));
 		offset += count;
 	}
-
 }
 
 void Registry::PrepareDrawUploadUniforms(bool drawBoundingBox)
 {
-	auto &renderCtx = Context().renderContext;
+	auto& renderCtx = Context().renderContext;
 
 	// Set transforms for instanced draw at offsets
 	auto prepareDrawBoundingBox = [&renderCtx, drawBoundingBox, this](uint32_t idx, const Transform& transform, MeshId meshId, int8_t submeshId) {
 		if (drawBoundingBox)
 		{
 			const L3DMesh& mesh = Game::instance()->GetMeshPack().GetMesh(static_cast<uint32_t>(meshId));
-			auto box = mesh.GetSubMeshes()[(mesh.GetNumSubMeshes() + submeshId) % mesh.GetNumSubMeshes()]->GetBoundingBox();
+			auto box            = mesh.GetSubMeshes()[(mesh.GetNumSubMeshes() + submeshId) % mesh.GetNumSubMeshes()]->GetBoundingBox();
 
 			glm::mat4 boxMatrix = glm::mat4(1.0f);
 			boxMatrix           = glm::translate(boxMatrix, transform.position + box.center());
-//			boxMatrix           = glm::rotate(boxMatrix, transform.rotation.x, glm::vec3(1.0f, 0.0f, 0.0f));
-//			boxMatrix           = glm::rotate(boxMatrix, transform.rotation.y, glm::vec3(0.0f, 1.0f, 0.0f));
-//			boxMatrix           = glm::rotate(boxMatrix, transform.rotation.z, glm::vec3(0.0f, 0.0f, 1.0f));
-			boxMatrix           = glm::scale(boxMatrix, transform.scale * box.size());
+			//			boxMatrix           = glm::rotate(boxMatrix, transform.rotation.x, glm::vec3(1.0f, 0.0f, 0.0f));
+			//			boxMatrix           = glm::rotate(boxMatrix, transform.rotation.y, glm::vec3(0.0f, 1.0f, 0.0f));
```

Comment:
This is my bad but the comments characters should be moved instead of appending the whole line